.gitignore

```diff
@@ -3,6 +3,7 @@
 .DS_Store
 /build/
 /e/build/
+.idea/
```

Comment:
You may add this to `$XDG_CONFIG_HOME/git/ignore`/`core.excludesFile` to avoid having to add it to each project you work on.  I don't mind that it is here -- the personal git ignore is just a better home for editor specific configs.

Reply:
:+1: